tests/integration/multi_table/test_hma.py

```diff
@@ -178,6 +181,43 @@ def get_custom_constraint_data_and_metadata():
 
 def test_hma_custom_constraint():
```

Comment:
What's the difference between `test_hma_custom_constraint` and `test_hma_custom_constraint_1_table`?

Reply:
I think he's testing that if the child table has the column name in the constraint, it doesn't get affected somehow by the parent table adding the constraint. Might be unnecessary to check